component/notification/src/main/java/org/exoplatform/social/notification/channel/template/WebTemplateProvider.java

```diff
@@ -273,10 +273,7 @@ protected boolean makeDigest(NotificationContext ctx, Writer writer) {
      */
     @Override
     public NotificationInfo getNotificationToStore(NotificationInfo notification) {
-      String activityId = notification.getValueOwnerParameter(SocialNotificationUtils.ACTIVITY_ID.getKey());
-      String parameterName = SocialNotificationUtils.POSTER.getKey();
-      String parameterValue = notification.getValueOwnerParameter(parameterName);
-      return SocialNotificationUtils.addUserToPreviousNotification(notification, parameterName, activityId, parameterValue);
+      return notification;
```

Comment:
Why don't you simply remove this function ? the method AbstractTemplateProvider.getNotificationToStore make exactly the same thing as you.